include/ginkgo/core/base/executor.hpp

```diff
@@ -1336,8 +1336,8 @@ class EnableDeviceReset {
 
     [[deprecated(
         "device_reset is no longer supported, call "
-        "cudaDeviceReset/hipDeviceReset manually")]] EnableDeviceReset(bool
-                                                                           device_reset)
+        "cudaDeviceReset/hipDeviceReset "
+        "manually")]] EnableDeviceReset(bool device_reset)
```

Comment:
This was formatted by the github actions. So, I believe this should be correct.

Reply:
No, I think it should not be touched in the beginning so the GitHub action should not check it.
clang-format can not concat two string, so if it is splited first then they can not recover it.
To ensure it is correct, you need to combine these two string and then run clang-format.

Reply:
Unfortunately, that is a bit hard now, as the changes to the file were introduced sometime back and it is hard to roll back only for this file without completely destroying the history, particularly because my `clang-format` seems to somehow format differently. I guess there are two options:

1. We have this version for now and fix it later when some other changes are introduced to this file.
2. You or someone else who has the same `clang-format` version can explicitly remove the new line and call `clang-format` on this file to restore it back to the original formatting.